app/workers/education_form/process10203_submissions.rb

```diff
@@ -29,7 +27,7 @@ def perform(
         saved_claims: {
           form_id: '22-10203'
         }
-      )
+      ).order('created_at')
```

Comment:
Since you are joining multiple tables, this 'created_at' that you are wanting to order by is ambiguous and is causing the CI check to fail. I think if you specify the specific table like` .order('TABLE_NAME.created_at')` it should work.

Reply:
updated